rocksdb_admin/admin_handler.cpp

```diff
@@ -609,7 +609,7 @@ void AdminHandler::async_tm_addDB(
   std::string err_msg;
   replicator::DBRole role = replicator::DBRole::SLAVE;
   if (request->__isset.db_role) {
-    if (request->db_role == "SLAVE") {
+    if (request->db_role == "SLAVE" || request->db_role == "FOLLOWER") {
```

Comment:
Can we Add DBRole::LEADER / FOLLOWER as well... so that we don't have to carryon the burden of using M/S terminology later on and doesn't becomes a mixed baggage... What that means is that it is either M/S or L/F ... Role...

Reply:
I want to keep this PR limited to adding the new state transitions. Adding the new role would lead to changes in multiple places which I want to avoid in the PR.